src/utils/templates.ts

```diff
@@ -154,7 +161,8 @@ importer.on({ action: "listCandidates" }, async ({ filters, nextPage }, {identif
 
   eventHandlersTemplate(
     contributionName: string,
-    contribution: { [k: string]: any }
+    contribution: { [k: string]: any },
+    identifier: string
```

Comment:
lint is complaining because this is unused, is there a missing piece of the diff or something?

Reply:
No, it's complaining because that param is only used in one place, but all of the methods need the same signature. I've added an underscore to quiet it down.